docs/static/keystore.asciidoc

```diff
@@ -121,11 +133,10 @@ To create a secrets keystore, use the `create`:
 bin/logstash-keystore create
 ----------------------------------------------------------------
 
-Creates the keystore in the directory defined by the `path.settings`
-configuration setting. This will also override an existing keystor setting. 
+Creates the keystore in the directory defined in the `path.settings` setting.
+This command also overrides an existing keystore setting. 
```

Comment:
The `create` command will actually overwrite the whole keystore (after a y/n prompt), and wipe out all the keys/secrets that were previously stored.